### ISSUE TITLE
Is this project dead ? 

### ISSUE THREAD

**Issue by Edensplash** [rrag/react-stockcharts#764]
I discover it but seems no update since 2 years ? 

I saw also gochart.com service based on that lib, is it still alive too ?

**Comment by yay** [comment]
Like you said it's been effectively dead for 2 years, but I'd say it's better then most free of charge financial libraries in active development. If you have a feature request or a bug report, it's not like you can expect free support anyway.

**Comment by varun2india89** [comment]
@yay Hello, 
Can you modification in this lib for additional cost?  I have some task in this lib, can you help ?

**Comment by yay** [comment]
@varun2india89 Not really. I'm not looking for work and wasn't implying I could help out for a fee. Just saying that I wouldn't hesitate using this library because it is no longer maintained.

**Comment by whidbey** [comment]
greate project I could say..

**Comment by markmcdowell** [comment]
We took a fork of this library about a year ago and we've been rebuilding, updating it, and we'll be supporting it.

See: https://github.com/reactivemarkets/react-financial-charts

Thanks!
